dino.icu.yaml

```
@@ -98,6 +98,11 @@ aryanpatel: # by https://github.com/aryanpatel142006
   type: CNAME
   value: cname.vercel-dns.com. # vercel
 
+baguette: # a blog
+  ttl: 600
+  type: CNAME
+  value: parkingturkeys.github.io/oui-oui-baguetti
```

You can't have a protocol or path in a CNAME record. You can choose which github pages site is deployed on this domain by changing your repository's settings.
```suggestion
  value: parkingturkeys.github.io.
```